tests/unit/test_project.py

```diff
@@ -251,6 +251,21 @@ def test_project_title_empty_invalid_name(yaml_loaded_data):
     assert "Invalid name for ROCK" in str(err.value)
 
 
+@pytest.mark.parametrize("entrypoint_service", ["hello"])
+def test_project_entrypoint_service_valid(yaml_loaded_data, entrypoint_service):
+    yaml_loaded_data["entrypoint_service"] = entrypoint_service
+    project = Project.unmarshal(yaml_loaded_data)
+    assert project.entrypoint_service == entrypoint_service
+
+
+@pytest.mark.parametrize("entrypoint_service", ["bye"])
+def test_project_entrypoint_service_invalid(yaml_loaded_data, entrypoint_service):
+    yaml_loaded_data["entrypoint_service"] = entrypoint_service
+    with pytest.raises(ProjectValidationError) as err:
+        Project.unmarshal(yaml_loaded_data)
+    assert "entrypoint_service used is invalid!" in str(err.value)
+
+
```

Comment:
could you also please test:
 - that the warning message is emitted
 - that the project loads well without this attribute. I say this because `entrypoint-service` should actually be part of ROCKCRAFT_YAML. this fixture should contain all the allowed fields, for the unmarshalling tests
 - when the value is empty